projects/frontend/cicd/build_data_pipelines.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# For installing the build prerequisites, consider `install_data_pipelines.sh` script.
+# This script is meant to be used, to quickly
+# rebuild and link the the @vdk/data-pipelines in an already bootstrapped development setup.
+###
+
+if ! which npm >/dev/null 2>&1 ; then
```

Comment:
Is this check really necessary? We already know what's in the runner image.

Reply:
The script is meant to be reused during local development as well, see #L6
```
###
 # For installing the build prerequisites, consider `install_data_pipelines.sh` script.
 # This script is meant to be used, to quickly
 # rebuild and link the the @vdk/data-pipelines in an already bootstrapped development setup.
 ###
```
In both cases it's good to have a message asserting on prerequisites known, to reduce on debugging complexity